tests/device_tests/test_passphrase_slip39_advanced.py

```diff
@@ -33,6 +34,21 @@ def test_128bit_passphrase(client):
     client.set_passphrase("TREZOR")
     address = btc.get_address(client, "Bitcoin", [])
     assert address == "1CX5rv2vbSV8YFAZEAdMwRVqbxxswPnSPw"
+    device.wipe(client)
```

Comment:
it's not necessary to wipe and restore device to try a new passphrase.

use the following code:
```
client.state = None
client.clear_session()
client.set_passphrase(...)
```